packages/common/src/content/compose.ts

```diff
@@ -255,6 +255,7 @@ export const getContentTypeIcon = (contentType: string) => {
     hubInitiative: "initiative",
     hubInitiativeTemplate: "initiative-template",
     hubPage: "browser",
+    hubProject: "briefcase",
```

Comment:
we won't be using this in opendata-ui, but since we're in here I thought I'd add it for hub-components